for-devs.md

```diff
@@ -27,10 +27,12 @@ This one might be obvious, but please keep in mind to ALWAYS `Ctrl+F` before ren
 
 ## Tested on...
 
-- Linux → Debian → Ubuntu → PopOS:
-  - My main system, should work 100% fine
+- Debian → Ubuntu → Pop! OS:
```

Comment:
should these have spaces next to the arrows? or is it just github mobile formatting

Reply:
They use periods to let you figure out if there is a space there or not, In this case; It's intentional accessibility.